example/50-seed-local.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   secretRef:
     name: seed-local
     namespace: garden
-  ingressDomain: <minikube-ip>.nip.io
+  ingressDomain: <local-kubernetes-ip>.nip.io
```

Comment:
Please change this in `hack/templates/resources/50-seed.yaml.tpl` because these examples are rendered by running `hack/generate-examples`.